config/console.php

```diff
@@ -1,27 +1,3 @@
 <?php
 
-use Yiisoft\Factory\Definitions\Reference;
-
-return [
-//    'app' => [
-//        'bootstrap' => [
-//            'queue' => 'queue',
-//        ],
-//    ],
-//    \PDO::class => Reference::to('pdo'),
-//    'pdo'       => [
-//        '__class'       => \PDO::class,
-//        '__construct()' => [
-//            'dsn'       => 'pgsql:dbname='.$params['db.name']
-//                .(!empty($params['db.host']) ? (';host='.$params['db.host']) : '')
-//                .(!empty($params['db.port']) ? (';port='.$params['db.port']) : ''),
-//            'username'  => $params['db.user'],
-//            'password'  => $params['db.password'],
-//            'options'   => [],
-//        ],
-//    ],
-//    \yii\mutex\Mutex::class => Reference::to('mutex'),
-//    'mutex'                 => [
-//        '__class' => \yii\mutex\MysqlMutex::class,
-//    ],
-];
+declare(strict_types=1);
```

Comment:
That looks suspicious. Why do we need an empty config that doesn't even return empty array?

Reply:
Didn't remove it because we will need it for console commands. I'll add empty array here for now.